src/multiversioning.zig

```diff
@@ -198,20 +198,20 @@ pub const MultiversionHeader = extern struct {
 
         count: u32 = 0,
 
-        releases: [past_releases_max]u32 = std.mem.zeroes([past_releases_max]u32),
-        checksums: [past_releases_max]u128 = std.mem.zeroes([past_releases_max]u128),
+        releases: [past_releases_max]u32 = @splat(0),
```

Comment:
I wonder if we should use `[past_releases_max]Release` here? Even if we do, that'll be follow-up!